public/css/components/sections/_main-preview.scss

```diff
@@ -23,3 +23,48 @@
 
   @include generate-player-sizing(100%);
 }
+
+
+.videojs-hero-overlay {
+   position: absolute;
+   text-align: center;
+   top: 37.5%;
+   left: 0;
+   width: 100%;
+   margin: 0;
+   padding: 0;
+   background-color: rgba(250, 252, 255, 0.95);
+   color: #757A7D;
+   font-family: "Lato", "Helvetica Neue", Helvetica, Arial, sans-serif;
+
+   .videojs-hero {
```

Comment:
We haven't been good about putting together an official SCSS style guide, but we've tried to keep nesting for just pseudo-elements. You don't have to worry about it in in this PR, we can take another pass after it's pulled in.